Cookbook/Technical-Documents/ApplicationDistribution.md

```diff
@@ -0,0 +1,24 @@
+# Application Distribution
+This document is for any Babylon iOS developers who wish to distribute our applications, or for any partners who wish to gain access to our applications.
+
+## How to gain access to iOS applications
+### External Babylon partners
+* If you are an external partner of Babylon, please speak with your Babylon contact who will arrange this for you
+
+### Babylon Employees
+* If you are a Babylon employee, please reach out to the current iOS support engineer(s) in the #ios slack channel
+* You will also need to have [App Center](https://github.com/babylonhealth/ios-playbook/blob/simon/cnsmr-3226/Cookbook/Technical-Documents/AppCenter.md) installed
```

Comment:
This document should not be intended for anyone other than iOS developers.

Instead I would focus on the process form this point of view:
```
 Requests to grant access must be through #ios channel and it must list each app they need access for. 
```
 You should list here which apps are eligible for each group. Keep in mind that Babylon meplyeees can have access to Testflight if they need to.

Reply:
@viorel15 by `which apps are eligible for each group` can you explain further what you mean by group here please? Is the `group` here relating to internal vs external?

Reply:
Yes, that is what I meant. Employees vs partners

Reply:
I've made some changes - please let me know your thoughts @viorel15